### PR TITLE
adding menu creation form + appending it to the page + NOW WITH EDITING!

### PR DESCRIPTION
**Pull request by blackcatwizard**
this should work. let me know if things break

### CODE REVIEW

**Comment by christophersutton** [comment]
Hey I think you squashed some stuff with your commits, though honestly I have no idea how. But the NewMenu view isn't showing any of the added items - looks like in one commit you added it in but removed the form, then another commit took it out and added form back in.

**Comment by blackcatwizard** [comment]
heck, i'll have to fix that in the morning

**Comment by blackcatwizard** [comment]
it seems to work on my machine at the moment, so maybe something is weird in my commit

File: src/pages/Menu/NewMenu.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+const onFormSubmit = event => {
+    
+}
+
+
+return(
+<div>
+    <div>
+        
+        <NewMenuItem
```
Comment (christophersutton) [inline src/pages/Menu/NewMenu.js:27]: check here for one commit

**Pull request merged**